examples/echo/README.md

```diff
@@ -0,0 +1,76 @@
+## Prerequisites
+
+- [Go](https://golang.org/doc/install)
+- [ngrok](https://ngrok.com/download)
+
+    ```bash
+    # start a HTTP tunnel on port 8080
+    ./ngrok http 8080
+    ```
+
+Attention! Ngrok tunneling should not be used for production applications and was used here only for demonstration ease of use.
+
+## Creating a LiveChat app
+
+1. Sign in to [Developer Console](https://developers.livechatinc.com/console/).
+2. Go to the [Apps](https://developers.livechatinc.com/console/apps/) section.
+3. Create a new app! You may use a blank template and add Authorization block yourself or use the webhooks template.
+4. Configure Authorization block. 
+	4.1 Choose the server-side as a client type. 
+	4.2 Add `$NGROK_PUBLIC_URL/oauth` to a redirect URI whitelist.
+	4.3 Copy client id, client secret, redirect URI. You will need them when creating a configuration file for the app.
+	4.4 Add `webhooks--all:rw` and `chats--all:rw` to requested scopes list.
+	4.5 Use `$NGROK_PUBLIC_URL/oauth` as a direct installation URL in the Marketplace authorization flow settings section.
+
+Attention! For production applications, you can skip point 4.5.
+Having troubles? Visit docs dedicated page for [createing LiveChat apps](https://developers.livechatinc.com/docs/getting-started/guides/#creating-livechat-apps).
+
+## Running this example
+
+- Clone the repository
+
+    ```bash
+    git clone https://github.com/livechat/lc-sdk-go.git
+    ```
+
+- In a terminal, navigate to `lc-sdk-go/examples/echo`
+
+    ```bash
+    cd lc-sdk-go/examples/echo
+    ```
+
+- Create a config file from the example
+
+    ```bash
+    mv config.example.json config.json
```

Comment:
config.example.json is tracked by git, `cp` would be better than `mv`

also, this won't work on Windows, please either mention that this tutorial is valid for Mac/Linux or add example command for the Windows as well :D